### ISSUE TITLE
Extra user in room

### ISSUE THREAD

**Issue by skug67** [mautrix/gmessages#6]
For some inexplicable reason a one-to-one SMS/MMS thread (it did include media, so presumably MMS) was added to matrix as a room with 4 users -- me, the gmessages bot, the correct gmessages userid (RightPhone), and another gmessages userid (WrongPhone).  As far as I can tell this is just a glitch in the internal representation and messages posted to the room going only to RightPhone and not WrongPhone.  And incoming texts from RightPhone show up as well.  But it is weird (and it also causes this chat to show up as a "Room" and not a "Person"  which is vaguely annoying)

**Comment by olmari** [comment]
While I am not touching any other aspects of the issue, with at least Element-web/desktop you do have the command `/converttodm` which ought to categorize it properly again for you.

**Comment by skug67** [comment]
Thanks.  Not sure what's going on under the hood and seems like there may a glitch which could use fixing.  But from a usability perspective that pretty much solves my problem.

**Comment by skug67** [comment]
Good idea.  Here's a log with all of the transactions referencing that room_id around the time it was set up.  @gmessages1_6:my.matrix.server is the user that the thread really belongs to.  @gmessages1_2:my.matrix.server is the extra user that doesn't belong in this thread (and, happily, does not otherwise actually seem to be connected to the thread).

(PS -- I did find another thread that was also converted to a group -- the thread for which @gmessages1_2 is actually the intended user; in that one the extra user is yet someone else and not @gmessages1_6.  Pretty much all of the other threads that I care about at all were handled correctly.  But admittedly, I haven't tried to do a comprehensive inventory of my whole SMS/MMS history of things like OTPs....)

[matrix.log](https://github.com/mautrix/gmessages/files/12215539/matrix.log)

**Comment by tulir** [comment]
Synapse logs aren't relevant, only bridge logs are. The latest commit logs the conversation info that the bridge sees whenever syncing
